### PR TITLE
new endpoint for periodic status/health checks

### PR DESCRIPTION
**Pull request by vesellov**

### CODE REVIEW

File: src/main/settings.py [hunk 1]
```diff
@@ -453,6 +453,9 @@
 if 'RPC_CLIENT_HEALTH_FILE' not in os.environ and ZENAIDA_GATE_HEALTH_FILENAME:
     os.environ['RPC_CLIENT_HEALTH_FILE'] = ZENAIDA_GATE_HEALTH_FILENAME
 
+ZENAIDA_GATE_HEALTH_CHECK_DOMAIN_NAME = getattr(params, 'ZENAIDA_GATE_HEALTH_CHECK_DOMAIN_NAME', 'domain-check.ai')
+ZENAIDA_GATE_HEALTH_CHECK_PERIOD = getattr(params, 'ZENAIDA_GATE_HEALTH_CHECK_PERIOD', 30)
```
Comment (ilhanmercan) [inline src/main/settings.py:457]: let's add this to params_example.py too

File: src/front/views.py [hunk 2]
```diff
@@ -424,6 +427,38 @@ def form_valid(self, form):
...
+        try:
+            latest_status = json.loads(cache.get(self.cache_key))
+        except:
+            latest_status = {}
+        if not latest_status:
+            latest_status = {
+                'epp': self.check_epp_status(),
+                # TODO: other status checks to be added here if required
+            }
+            cache.set(self.cache_key, json.dumps(latest_status), timeout=settings.ZENAIDA_GATE_HEALTH_CHECK_PERIOD)
```
Comment (ilhanmercan) [inline src/front/views.py:457]: I am curious if we need to convert the latest status to JSON here? I think it should be working without that too

File: src/main/settings.py (hunk 1 above)
Comment (vesellov) [reply src/main/settings.py:457]: fair point! done!

File: src/front/views.py (hunk 2 above)
Comment (vesellov) [reply src/front/views.py]: indeed.! updated!

**Review by ilhanmercan — APPROVED** [review]

**Pull request merged**